src/Workspaces/Core/Portable/Diagnostics/DiagnosticData.cs

```diff
@@ -303,7 +307,7 @@ public static LinePositionSpan GetLinePositionSpan(DiagnosticDataLocation dataLo
 
         public static TextSpan GetTextSpan(DiagnosticDataLocation dataLocation, SourceText text)
```

Comment:
i rename this method in a sibling pr to indicate that it's only getting the unmapped textspan.